src/components/templates/Audience/Audience.tsx

```diff
@@ -138,14 +132,20 @@ const requiredAuditoriumSize = (
   return size;
 };
 
+const burningReactions = EmojiReactions.filter(
+  (reaction) =>
+    reaction.type !== EmojiReactionType.boo &&
+    reaction.type !== EmojiReactionType.thatsjazz
+);
+
 // Note: This is the component that is used for the Auditorium
 export const Audience: React.FunctionComponent = () => {
   const venueId = useVenueId();
-  const { user, profile } = useUser();
+  const { userWithId, profile } = useUser();
   const venue = useSelector(currentVenueSelectorData);
   const { recentVenueUsers } = useRecentVenueUsers();
 
-  const userUid = user?.uid;
+  const userUid = userWithId?.id;
```

Comment:
I've seen so many times that we tend to only import user to retrieve his id. Is it worth to just return userId var from the hook?

Reply:
Yeah, that seems like a reasonable change to make. Might be OOS of this PR though, as to do it properly we should clean up all the places in the codebase that are using this pattern at the same time.

Reply:
Added to `useUser` in https://github.com/sparkletown/sparkle/pull/1323/commits/e4fae8398c59a12504f0f0322c1d20bf0ae5895a

Fixed in `Audience` in https://github.com/sparkletown/sparkle/pull/1323/commits/7f0afea8265473ef7ec84c8cb83041973281a3c1

Refactoring the rest of the codebase in this PR is OOS